github/repos.go

```diff
@@ -562,6 +562,8 @@ type PullRequestReviewsEnforcement struct {
 	DismissalRestrictions DismissalRestrictions `json:"dismissal_restrictions"`
 	// Specifies if approved reviews are dismissed automatically, when a new commit is pushed.
 	DismissStaleReviews bool `json:"dismiss_stale_reviews"`
+	// Specifies if an approved review is required in pull requests including files with a designated code owner.
```

Comment:
To maintain the same style as the current API, this should be `RequireCodeOwnerReviews specifies...`

Reply:
(That is a hard requirement for top level identifiers, but for struct fields, I think both ways are acceptable. This is currently consistent with the documentation for fields above.)